.github/dependabot.yml

```diff
@@ -41,24 +41,6 @@ updates:
         update-types:
           - "version-update:semver-patch"
           - "version-update:semver-minor"
-  - package-ecosystem: terraform
```

Comment:
@cmbuckley Is there any reason this is happening?

Reply:
Ah a weird race condition where my current branch doesn't have this module from main

Reply:
Maybe not. I'm confused

Reply:
Your branch is behind main, and it doesn't contain #58. This won't cause an issue, but you can rebase the branch if you want to clear up the confusion - you'll notice the dependabot commit isn't included because it was already done in e19098adfd9d74080826e40d46b221477f2a3b62.